src/Controller/ConfigDataObjectController.php

```diff
@@ -65,9 +65,7 @@ public function __construct(PreviewService $previewService)
      *
      * @param Request $request
      *
-     * @return JsonResponse
-     *
-     * @throws \Exception
+     * @return JsonResponse|null
      *
```

Comment:
```suggestion
     * @throws \Exception
```
$this->checkPermission(self::CONFIG_NAME); could throw an exception